File: Shared/Blog.Contracts/V1/Responses/TagsResponses/PagedTagsResponse.cs
using System.Collections.Generic;

namespace Blog.Contracts.V1.Responses.TagsResponses
{
    /// <summary>
    /// Paged tags response.
    /// </summary>
    public class PagedTagsResponse
    {
        /// <summary>
        /// Gets or sets the tags.
        /// </summary>
        /// <value>
        /// The tags.
        /// </value>
        public IList<TagResponse> Tags { get; set; }

        /// <summary>
        /// Gets or sets display type.
        /// </summary>
        public string DisplayType { get; set; }

        /// <summary>
        /// Gets or sets page info.
        /// </summary>
        public PageInfoResponse PageInfo { get; set; }
    }
}

File: Shared/Blog.FSharp.Contracts/V1/Responses/TagsResponses/PagedTagsResponse.fs
namespace Blog.FSharp.Contracts.V1.Responses.TagsResponses

open System.Collections.Generic
open Blog.FSharp.Contracts.V1.Responses

/// <summary>
/// Paged tags response.
/// </summary>
type PagedTagsResponse() =
    /// <summary>
    /// Gets or sets the tags.
    /// </summary>
    member val Tags: IList<TagResponse> = List<TagResponse>() :> IList<TagResponse> with get, set

    /// <summary>
    /// Gets or sets display type.
    /// </summary>
    member val DisplayType: string = "" with get, set

    /// <summary>
    /// Gets or sets page info.
    /// </summary>
    member val PageInfo: PageInfoResponse = PageInfoResponse() with get, set

